optaplanner-docs/src/main/asciidoc/index.adoc

```diff
@@ -1,7 +1,7 @@
 = OptaPlanner User Guide
 The OptaPlanner Team <https://www.optaplanner.org/community/team.html>
 :doctype: book
-:title-logo-image: image:Chapter-Planner_introduction/optaPlannerLogo.png[align="center"]
+:title-logo-image: image:PlannerIntroduction/optaPlannerLogo.png[align="center"]
```

Comment:
Because the images dirs haven't changed yet, this image isn't found and a warning is shown in the "mvn clean install" log.
I've fixed this by reverting this line for now. The image dirs need to change too, of course, but let's first see if we can go to relative images.